articles/openshift/oshift-ref-kibana-dashboards.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Explanation of the custom dashboards we provide with the aggregated logging kibana | UKCloud Ltd
+description: Explains what our custom dashboards in Kibana show and how this data can be used.
+services: openshift
+author: Kieran O'Neill
+reviewer:
+lastreviewed: 
+
+toc_rootlink: Reference
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Explanation of the custom dashboards we provide with the aggregated logging kibana
+toc_fullpath: Reference/oshift-ref-kibana-dashboard.md
+toc_mdlink: oshift-ref-kibana-dashboard.md
+---
+
+# Explanation of the custom dashboards we provide with the aggregated logging kibana
+
+## Overview
+
+This documentation contains an explanation of the two custom dashboards we configure with the Kibana deployment in your cluster, these are called "Hosts Logging" and "Response Codes". It explains what data is being shown and what information you can derive from this.  
+
+## Intended audience
+
+The "Hosts Logging" dashboard is available in 3.9-3.11 clusters and the "Response Codes" dashboard is available in 3.11 clusters.
+
+## How to Access
+
+Open the kibana route, depending on your cluster version it will have the following format: `https://kibana.xxxx-xxxxx.region.cna.ukcloud.com` or `https://kibana.xxx-xxx-xx-xxxxx.region.cna.ukcloud.com` 
```

Comment:
Open the Kibana route [is route the right word here? It doesn't sound right to me]. Depending on your cluster version, this will have the following format...

Reply:
Route is an OpenShift term for a service exposed outside the cluster. I could also use URL but I think route will be clearer to customers.

Reply:
OK. If it's a term that customers will understand, keep it.